schema/primitive.proto

```diff
@@ -4,6 +4,7 @@ package iroha.protocol;
 enum RolePermission {
    // Command Permissions
     can_append_role = 0;
+    can_detach_role = 29;
```

Comment:
Maybe move it to the end?

Reply:
But the permission is related to commands